Playbooks/playbook-Protectwise_Demo_Playbook.yml

```diff
@@ -8,6 +8,46 @@ description: Displays information for all events since 2016-06-17 and all observ
   limit .
 tasks:
 - id: "1"
+  taskid: 1bf3408d-bfcc-4030-8406-126c9514ee5b
```

Comment:
Should a playbook with this name be in our system ?
I think that we should only have production playbooks in the content